.github/workflows/upstream.yml

```diff
@@ -11,7 +11,7 @@ on:
         required: true
         default: "master"
       db-branch:
-        description: "DB Branch"
+        description: "Not used, left for backward compatibility"
```

Comment:
left for another week or two as removing it will break existing PRs and release branches (by default, if PR has no changes to workflow, workflow from master is used and non-existing input will fail the check).

Reply:
@abalakh `pr_id` is required, can it break existing PRs and release branches?

Reply:
Good catch. Yep, should have been optional